src/fiat-connect-client.ts

```diff
@@ -417,6 +420,9 @@ function handleError<T>(error: unknown): Result<T, ResponseError> {
   if (error instanceof ResponseError) {
     return Result.err(error)
   } else if (error instanceof Error) {
+    if (error.name === 'AbortError') {
+      return Result.err(new ResponseError(error.name))
+    }
```

Comment:
Considered making this return `ResponseError | AbortError`, but that would mean breaking changes for every endpoint, so just make `error.message` to be "AbortError", which clients can check for